src/cmd/singularity/cli/actions.go

```diff
@@ -208,6 +209,11 @@ func execStarter(cobraCmd *cobra.Command, image string, args []string, name stri
 		engineConfig.SetImage(abspath)
 	}
 
+	if Nvidia {
+		NvidiaBindPaths := nvidiautils.GetNvidiaBindPath(buildcfg.SINGULARITY_CONFDIR)
```

Comment:
```
NvidiaBindPaths, err := nvidiautils.GetNvidiaBindPath(buildcfg.SINGULARITY_CONFDIR)
if err != nil {
    sylog.Fatalf("Unable to capture nvidia bind points: %v", err)
}
```

Reply:
Maybe it should be a warning instead of a failure though?  That is the way 2.x works.